spec/views/claim_reviews/show/review_page_claim_details_spec.rb

```diff
@@ -27,7 +27,7 @@ def claim_path_for(*)
     end
 
     let(:claim) do
-      create :claim, claim_details: "wut\r\nwut", other_known_claimant_names: "Johnny Wishbone\r\nSamuel Pepys"
+      create :claim, claim_details: "wut\r\nwut", other_known_claimants: true, other_known_claimant_names: "Johnny Wishbone\r\nSamuel Pepys"
```

Comment:
Not one for this spec - but there was nowhere else to add a comment.  You need to prove that the new attribute makes it to the API - there should already be a spec for that - but you will need to add this to it